docs/vertiq_testing_tool/vertiq_testing_tool_start_guide.rst

```diff
@@ -0,0 +1,116 @@
+It provides a streamlined interface to control your module using ESC, Voltage, Velocity, and PWM inputs.
+
+
+********
+Download
+********
+Currently, Vertiq Testing Tool is only available on Windows (Linux coming soon).
+You can download Vertiq Testing Tool on our `Support Page <https://www.vertiq.co/support>`_.
+
+
+**********************
+Hardware Configuration
+**********************
+Please follow the :ref:`Control Center documentation <connection_guide>` to connect your module to your computer.
+
+
+********
+Overview
+********
+Once downloaded, extract and run the Vertiq Testing Tool executable. When running this application for the first time, 
+you may encounter a Windows Defender SmartScreen warning.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/windows_defender.png
+
+Click on 'More info'.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/windows_defender_more_info.png
+
+Then click 'Run anyway'.
+
+.. image:: ../_static/vertiq_testing_tool_pictures/welcome_screen.png
+
+Once the application is loaded, you will be presented with the welcome screen. Follow the instructions on this screen to connect your module.
+
+
+.. image:: ../_static/vertiq_testing_tool_pictures/esc_input.png
+
+#. The :gold:`Tabs` section are the different tabs that you can switch to in order to spin your module.
+
+#. The :green:`Plotting` section contains a live plot in the upper section and a dashboard with widgets to control the live plot in the lower section.
+
+#. The :purple:`Control Panel` section contains widgets to configure specific parameters on your module. The Control Panel on each tab contains different widgets. 
+
+In this ESC Input tab, the Control Panel contains the same parameters that need to be configured as the example in :ref:`Control Center <flight_controller_config_with_control_center>`.
+The only difference is that these parameters are in all on the same tab, as opposed to different tabs in Control Center.
+
+==================
+Continuous command
+==================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/esc_input_command_widget.png
+
+A notable feature in the ESC Input tab is the 'Continuous command' at the bottom of the Control Panel. This feature is disabled by default, which means you need to click the 'Command ESC' button every time you want to send an ESC command.
+When this feature is enabled, the ESC command will be continuously sent at the 'Command rate' with 10 Hz being the default value. The 'Command rate' can be adjust to send the ESC command at different rates. 
+You can also use the slider to adjust the ESC input value.
+
+
+================================
+Live Plotting and Active metrics
+================================
+
+.. image:: ../_static/vertiq_testing_tool_pictures/voltage_input_command_example.png
+One main feature of Vertiq Testing Tool is the ability to plot any supported metric. For example, let's command 3 volts to the module using the Control Panel in the Voltage Input tab.
```

Comment:
what is a, "Supported metric?" I think you'll need to elaborate on the connection between metric and IQUART endpoints.